ci/build.sh

```diff
@@ -93,6 +93,13 @@ x86_64-unknown-freebsd \
 x86_64-unknown-linux-gnu \
 x86_64-unknown-linux-musl \
 x86_64-unknown-netbsd \
+armv7-wrs-vxworks \
```

Comment:
These are in the incorrect list of targets. You need to put them in the `RUST_NIGHTLY_LINUX_TARGETS` variable.

Reply:
Hi gnzlbg,

I have moved these vxworks targets to RUST_NIGHTLY_LINUX_TARGETS.

Thanks,
Baoshan